core/jobs/job_utils.py

```diff
@@ -102,8 +102,13 @@ def get_model_class(kind: Optional[str]) -> Type[datastore_services.Model]:
     # separate workers and some parts of the jobs are probably not available
     # to all the workers.
     models.Registry.get_all_storage_model_classes()
-    return datastore_services.Model._lookup_model( # pylint: disable=protected-access
-        kind
+
+    # Here we use cast because _lookup_model returns Optional[Model] or a less
```

Comment:
Is the main issue the Optional part of this? If so then can we handle it using an assertion that it's not None?